conftest.py

```diff
@@ -1,12 +1,18 @@
-import os
 import ConfigParser
-import pytest
-
+import os
+import shutil
+import sys
+import time
 from foxpuppet import FoxPuppet
-from helper_prefs import set_prefs
+import pytest
+from selenium.webdriver import Firefox, FirefoxProfile
+from selenium.webdriver.firefox.firefox_binary import FirefoxBinary
+from helper_prefs import set_prefs # noqa
+from os_handler import OSHandler
```

Comment:
os_handler - this is a convenience module in place to support OSX paths until we are fully dockerized